prac_05/wimbledon.py

```diff
@@ -24,22 +24,3 @@ def process_data(data):
             champions[champion] = 1
         countries.add(country)
     return champions, sorted(countries)
-
-def display_champions(champions):
-    print("Wimbledon Champions:")
-    for champion, wins in champions.items():
-        print(f"{champion} {wins}")
-
-def display_countries(countries):
-    print("\nThese", len(countries), "countries have won Wimbledon:")
-    print(", ".join(countries))
-
-def main():
```

Comment:
main() function could be moved to the top of the program code so that the readers of the code can see what the program does right away.